Documents/Management/WeeklyReports/weeklyReportThirdWeek.md

```diff
@@ -0,0 +1,50 @@
+# ThirdWeeklyReport
+
+[Cumulative Weekly Reports](mainWeeklyReport.md)
+
+## Week 3 (25-29 March 2024)
+
+The third week was devoted to managing each component with LEDs. The team discussed the project and how we would achieve it without the hardware breaking down. The technical specifications document was completed. The team also continued to reverse engineer the existing software with LEDs to try and implement all the functionality of the project.
+
+## This week's topics
+
+1. Manage each component with LEDs.
+2. Finish writing technical specifications.
+3. Continue reverse engineering existing software with LEDs.
+4. Attempt to implement project functionality without hardware but with LEDs.
+
+## Achievements
+
+1. Manage each component with LEDs
+   - during this week, the team managed to simulate the functionality of each component with LEDs.
+   - The team managed to simulate the functionality of the electromagnet, the buzzer, and the NFC antenna.
+
+2. Documents
+   - The technical specifications were completed.
+   - The test plan is well advanced.
+   - The user manual has been continued with the information available to us.
+
+3. Hardware components
+   - The team didn't manage to solve the hardware problems, so they decided to use LEDs to simulate the functionality of the project.
+
+4. Reverse engineering
+   - The team continued to reverse-engineer the existing software using LEDs to test the project's functionality.
+
+## Challenges
+
+The difficulties encountered during this week were mainly due to the hardware components. The team had trouble with the hardware components, as the components did not work as expected. The team tried to solve the problems without success and decided to use LEDs to simulate the functionality of the project.
+
+## Next week's plan
+
+1. Documents
+   - Continue the test plan.
+   - Continue the user manual.
+
+2. Hardware components
+   - The team will have a new battery to test the project's functionality.
```

Comment:
Could you please explain a little bit the part with the battery. Where does the battery came from ?